### PR TITLE
Add Java example in "get started"

### LINKED ISSUE #267: Java example in "get started" is missing
java got missing from get started at [https://coala.io/#/home#installation](https://coala.io/#/home#installation)
I think that get started for Java should be added.

### PR DESCRIPTION
**Pull request by fatma-mohamed**
Fixes #267   

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally.

### CODE REVIEW

**Comment by bekicot** [comment]
@fatma-mohamed Squash your commit.
`git reset --soft HEAD~2 && git commit`

File: resources/js/snippets.js [hunk 1]
```diff
@@ -15,5 +15,6 @@ var snippets = {
     "Ruby" : "ruby.md",
     "Typescript" : "typescript.md",
     "YAML" : "yaml.md",
-    "Lua" : "lua.md"
+    "Lua" : "lua.md",
+    "Java" : "java.md"
```
Comment (bekicot) [inline resources/js/snippets.js]: Put `"Java" : "java.md"` before `"JavaScript" : "javascript.md"`, so  it will  have chronological order.

**Comment by palash25** [comment]
Actually the 1st line of your description `java.md: Add java code snippet` should be your shortlog.

Replace your current shortlog with this one the rest looks fine. :smile:

**Comment by palash25** [comment]
Looks good to me @fatma-mohamed :+1:

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
ack ef25120

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**